Python/Product/Analysis/Parsing/CodeFormattingOptions.cs

```diff
@@ -236,6 +236,11 @@ public sealed class CodeFormattingOptions {
         [CodeFormattingDefaultValue(true)]
         public bool BreakMultipleStatementsPerLine { get; set; }
 
+        [CodeFormattingExample("foo.__bar", "foo._A__bar")]
```

Comment:
Does this end up showing up in tools->options->text editor->python->formatting ?  We don't really want it to be there, but we probably don't need all of the attributes if it doesn't.